server/events/vcs/common/common.go

```diff
@@ -43,3 +61,10 @@ func min(a, b int) int {
 	}
 	return b
 }
+
+func max(a, b int) int {
```

Comment:
As of golang 1.21 (and Alantis specifies go 1.22 in its go.mod), there is a builtin function call `max()` so this is not necessary. (A subsequent unrelated PR should remove `min()` from this file as well)